### PR TITLE
DCOS-38822: NodesTable / CPU Columns

### PR DESCRIPTION
**Pull request by juliangieseke**
## Testing

Change this line in `NodesTableContainer.js`

```diff
-import NodesTable from "../../../components/NodesTable";
+import NodesTable from "../../../components/NodesTable-new";
```

After that please take a look at the nodes table and see if the health column looks right and sorts correctly

## Trade-offs

- CPU percentage and resource bar are two different columns, once we have finished DCOS-39147, we can merge them and hide the bar on smaller screens

## Dependencies

None

### CODE REVIEW

File: plugins/nodes/src/js/columns/NodesTableCPUColumn.tsx [hunk 1]
```diff
@@ -7,8 +7,7 @@ import { IWidthArgs as WidthArgs } from "#PLUGINS/nodes/src/js/types/IWidthArgs"
 import { SortDirection } from "plugins/nodes/src/js/types/SortDirection";
 
 export function cpuRenderer(data: Node): React.ReactNode {
-  // TODO: DCOS-38822
-  return <span>{data.get("used_resources").cpus.toString()}</span>;
+  return `${data.getUsageStats("cpus").percentage}%`;
```
Comment (DanielMSchmidt) [inline plugins/nodes/src/js/columns/NodesTableCPUColumn.tsx]: Funny how template strings lead to us writing `${}` instead of `+` 😆

**Comment by juliangieseke** [comment]
sidenote: would be perfectly fine to block this PR because of its missing description, I simply forgot adding one 😟

File: plugins/nodes/src/js/columns/NodesTableCPUColumn.tsx [hunk 2]
```diff
@@ -7,8 +7,7 @@ import { IWidthArgs as WidthArgs } from "#PLUGINS/nodes/src/js/types/IWidthArgs"
...
 }
 export function cpuSorter(data: Node[], sortDirection: SortDirection): Node[] {
   // TODO: DCOS-38822
```
Comment (DanielMSchmidt) [inline plugins/nodes/src/js/columns/NodesTableCPUColumn.tsx]: There is a comment with this ticket number missing and the sorting is inverse of what the old implementation provides

Comment (juliangieseke) [reply plugins/nodes/src/js/columns/NodesTableCPUColumn.tsx]: hm, you're right, not the question is: was the old one intentional or was it a bug? 🤔

Comment (DanielMSchmidt) [reply plugins/nodes/src/js/columns/NodesTableCPUColumn.tsx]: No, unintentional

Comment (DanielMSchmidt) [reply plugins/nodes/src/js/columns/NodesTableCPUColumn.tsx]: It seems like github does not allow me to comment on the right side, but only here

Comment (juliangieseke) [reply plugins/nodes/src/js/columns/NodesTableCPUColumn.tsx]: after slack discussion: we keep it as is 👍

**Comment by juliangieseke** [comment]
removed todo comment 👍  @nLight @DanielMSchmidt please revisit 🙇

**Pull request merged**